incubator/spark/templates/NOTES.txt

```diff
@@ -0,0 +1,14 @@
+Getting Started:
+
+1. Get the Spark URL to visit by running these commands in the same shell:
+**** NOTE: It may take a few minutes for the LoadBalancer IP to be available.                                                                 ****
+****       You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "webui-fullname" . }}' ****
```

Comment:
With this line the `****`'s end up looking weird. In other charts we've removed it, in case you want to do it here too.